### PR TITLE
make avatar clickable

### PR DESCRIPTION
**Pull request by blackprince001**
just a minor quality of life feat

### CODE REVIEW

File: client/app/components/post-people.tsx [hunk 1]
```diff
@@ -23,7 +24,7 @@ function PostPeople({ post }: Props) {
 
 			<ul>
 				{people.map((person) => (
-					<li key={person.id}>
```
Comment (blackmann) [inline client/app/components/post-people.tsx:26]: Please maintain the li

File: client/app/routes/discussions_.$id.tsx [hunk 2]
```diff
@@ -144,8 +144,10 @@ export default function Discussion() {
 
 					<div className="flex gap-2 mt-4">
 						{user ? (
-							<>
-								<Avatar name={user?.username || ""} />
+							<> 
+                                <Link to={`/p/${user.username}`}>
+								    <Avatar name={user?.username || ""} />
+                                </Link>
```
Comment (blackmann) [inline client/app/routes/discussions_.$id.tsx]: Install biome extension and apply formatting

File: client/app/components/post-item.tsx [hunk 3]
```diff
@@ -117,7 +117,9 @@ function PostContent({ full, post, active, level, limit }: PostContentProps) {
 			<div className="flex flex-col items-center">
 				{full && (
 					<div className="mb-2">
-						<Avatar name={post.user.username} />
+                        <Link to={`/p/${post.user.username}`}>
+						    <Avatar name={post.user.username} />
+                        </Link>
```
Comment (blackmann) [inline client/app/components/post-item.tsx]: Apply formatting

**Review by blackmann — CHANGES_REQUESTED** [review]

File: client/app/components/post-people.tsx (hunk 1 above)
Comment (blackprince001) [reply client/app/components/post-people.tsx:26]: got it

**Review by blackmann — APPROVED** [review]

**Pull request merged**